lib/WWW/Mechanize.pm

```diff
@@ -90,7 +90,8 @@ traverse.
     $mech->back();
 
 If you want finer control over your page fetching, you can use
-these methods. C<follow_link> and C<submit_form> are just high
+these methods. C<L<< follow_link()|"$mech->follow_link(...)" >>>
```

Comment:
Section links should include a preceding `/`, like `L</"$mech->follow_link(...)">`. Linking sections without the `/` is deprecated.

Reply:
Yes, I thought that. But then I looked at what we're doing everywhere, and none of them have the slash. I think we should add them to all the links, not just this one.

Reply:
I'd agree that they all should be changed. Up to you if you want to include that with this PR.

Reply:
@simbabque let me know how you want to handle this?

Reply:
I'll update it today.